docs/TESTS.md

```diff
@@ -6,6 +6,26 @@ Choose your operating system:
 * [Mac OS X](#mac-os-x)
 * [Linux](#linux)
 
+## Understanding Skips
+
+The `skip` parameter instructs the test suite to not run a test. This is commonly used to avoid running tests of unimplemented functionality, so you can focus on the part you are currently working on.
+
+The test run summary will list the number of skipped tests.
+For example :
+
+```
+❯ pub run test
+00:01 +4 ~3 -1: Some tests failed.
+```
+
+`+` describes the number of tests passed. In this example, 4 tests are passed.
+
+`~` describes the number of tests skipped. In this example, 3 tests are skipped.
+
+`-` describes the number of tests failed. In this case, there is only one test that failed.
+
```

Comment:
```suggestion

The `skip` parameter instructs the test suite to not run a test. This is commonly used to avoid running tests of unimplemented functionality, so you can focus on the part you are currently working on.

```